astropy/timeseries/periodograms/lombscargle/implementations/cython_impl.pyx

```diff
@@ -26,10 +26,10 @@ def lombscargle_cython(t, y, dy, frequency, normalization='standard',
 
     Parameters
     ----------
-    t, y, dy : array_like  (NOT astropy.Quantities)
```

Comment:
again move `NOT astropy.Quantites` to text.

Reply:
✅